Title: Status Bar On All Monitors
Author: Eric Pruitt (https://www.codevat.com/, https://github.com/ericpruitt/)

With this patch, the status bar is shown on all monitors even when they are not
focused.

This patch depends on "dwm-00-systray.diff".

diff --git dwm.c dwm.c
index 4933a1f..e987f92 100644
--- dwm.c
+++ dwm.c
@@ -792,11 +792,9 @@ drawbar(Monitor *m)
 		stw = getsystraywidth();
 
 	/* draw status first so it can be overdrawn by tags later */
-	if (m == selmon) { /* status is only drawn on selected monitor */
-		drw_setscheme(drw, scheme[SchemeNorm]);
-		sw = TEXTW(stext) - lrpad / 2; /* no right padding so status text hugs the corner */
-		drw_text(drw, m->ww - sw - stw, 0, sw, bh, lrpad / 2 - 2, stext, 0);
-	}
+	drw_setscheme(drw, scheme[SchemeNorm]);
+	sw = TEXTW(stext) - lrpad / 2; /* no right padding so status text hugs the corner */
+	drw_text(drw, m->ww - sw - stw, 0, sw, bh, lrpad / 2 - 2, stext, 0);
 
 	resizebarwin(m);
 	for (c = m->clients; c; c = c->next) {
@@ -2209,9 +2207,13 @@ updatetitle(Client *c)
 void
 updatestatus(void)
 {
+	Monitor *m;
+
 	if (!gettextprop(root, XA_WM_NAME, stext, sizeof(stext)))
 		strcpy(stext, "dwm-"VERSION);
-	drawbar(selmon);
+	for (m = mons; m; m = m->next) {
+		drawbar(m);
+	}
 	updatesystray();
 }
 
